godel_msgs/action/ProcessPlanning.action

```diff
@@ -0,0 +1,19 @@
+# Goal
+int32 GENERATE_MOTION_PLAN_AND_PREVIEW=1
+int32 PREVIEW_TOOL_PATH=2
```

Comment:
This can totally be in a separate PR, but we should probably remove `PREVIEW_TOOL_PATH`: I don't think it does anything anymore. It's an artifact from before we even had a robot in Godel.